packages/components/time-picker/src/utils.ts

```diff
@@ -88,3 +89,9 @@ export const makeList = (total: number, method?: () => number[]) => {
   }
   return arr
 }
+
+export const dayOrDaysToDate = (dayOrDays: DayOrDays): Date | [Date, Date] => {
```

Comment:
```suggestion
export const dayOrDaysToDate = (dayOrDays: DayOrDays): DayOrDays => {
```

Reply:
This might be incorrect. You mean `DateOrDates`, right?